stackstorm-jinja/pack.yaml

```diff
@@ -0,0 +1,8 @@
+name: jinja
+description: Jinja template engine
+keywords:
+ - jinja
+ - template
+version: 0.1.0
+author: StackStorm dev
```

Comment:
Pretty sure I've picked that up from one of our packs. CircleCI pack, if I have to guess.